aptos-move/aptos-gas-schedule/src/gas_schedule/move_stdlib.rs

```diff
@@ -40,7 +40,6 @@ crate::gas_schedule::macros::define_gas_parameters!(
         [bcs_serialized_size_per_byte_serialized: InternalGasPerByte, { RELEASE_V1_18.. => "bcs.serialized_size.per_byte_serialized" }, 36],
         [bcs_serialized_size_failure: InternalGas, { RELEASE_V1_18.. => "bcs.serialized_size.failure" }, 3676],
 
-        [mem_swap_base: InternalGas, { RELEASE_V1_22.. => "mem.swap.base" }, 367],
-        [mem_swap_per_abs_val_unit: InternalGasPerAbstractValueUnit, { RELEASE_V1_22.. => "mem.swap.per_abs_val_unit"}, 14],
+        [mem_swap_base: InternalGas, { RELEASE_V1_22.. => "mem.swap.base" }, 1500],
```

Comment:
@igor-aptos FYI I've bumped the base cost quite a bit. The original one was too low -- it should be at least 2x that of `write_ref`.